src/views/settings/Applet/AppletHost/AppletHostList.vue

```diff
@@ -93,7 +97,7 @@ export default {
         hasExport: false,
         hasImport: false,
         onCreate: () => {
-          this.$refs.table.onCreate({ type: 'windows', category: 'host', platform: 'RemoteAppHost' })
+          this.$refs.table.onCreate({ query: appletRouteQuery })
         }
       }
     }
```

Comment:
The code has no obvious inconsistencies, problems or optimization opportunities based on the knowledge cutoff of September 2021. It is currently in good shape with regards to functionality and structure. If you have any specific areas that need attention or improvements, please feel free to ask!